### PR TITLE
graceful shutdown process added

### PR DESCRIPTION
**Pull request by DarKnight24**

### CODE REVIEW

File: disk/run.go [hunk 1]
```diff
@@ -32,14 +32,14 @@ var (
 	diskInfo  map[string]PartitionInfo
 	partition []string
 	logFile   *os.File
-	conf      *Config
 )
 
 // Disk is driver funcion for the health_monitor to monitor disk
 func Disk(status <-chan bool, wg *sync.WaitGroup) {
 	defer wg.Done()
 	var (
```
Comment (viyatb) [inline disk/run.go:40]: Why not declare all these globally?

File: live/run.go [hunk 2]
```diff
@@ -20,14 +20,14 @@ type Status struct {
...
 	liveStatus Status
 	logFile    *os.File
-	conf       *Config
 )
 
 // Live is the driver function of this module for monitor
 func Live(status <-chan bool, wg *sync.WaitGroup) {
 	defer wg.Done()
 	var (
 		logFileName = path.Join(setup.ConfigVars.HomeDir, "live.log")
```
Comment (viyatb) [inline live/run.go:29]: Same here.

File: setup/db.go [hunk 3]
```diff
@@ -3,18 +3,30 @@ package setup
...
+var (
+	Database  *sql.DB
+	DBLogFile *os.File
+)
 
 func dbInit() {
-	var DBDriver string
-	var err error
+	var (
+		DBDriver    string
```
Comment (viyatb) [inline setup/db.go:21]: Ditto.

File: disk/run.go (hunk 1 above)
Comment (DarKnight24) [inline disk/run.go:40]: Those variables are not required outside the scope of the function

**Pull request merged**